keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -219,6 +219,33 @@ def _build_shear_y_transform_matrix(shear_y):
     def augment_labels(self, labels, transformations, **kwargs):
         return labels
 
+    def augment_segmentation_masks(
+        self, segmentation_masks, transformations, **kwargs
+    ):
+        x, y = transformations["shear_x"], transformations["shear_y"]
+
+        if x is not None:
+            transforms_x = self._build_shear_x_transform_matrix(x)
+            segmentation_masks = preprocessing.transform(
+                images=segmentation_masks,
+                transforms=transforms_x,
+                interpolation=self.interpolation,
```

Comment:
We should use `nearest` interpolation for segmentation masks

Reply:
Done.